### ISSUE TITLE
Server Crash (MultiPaper)

### ISSUE THREAD

**Issue by TMKCRepo** [PureGero/MultiPaper#119]
[22:32:47] [Paper Watchdog Thread/ERROR]: If you are unsure or still think this is a Paper bug, please report this to https://github.com/PaperMC/Paper/issues
[22:32:47] [Paper Watchdog Thread/ERROR]: Be sure to include ALL relevant console errors and Minecraft crash reports
[22:32:47] [Paper Watchdog Thread/ERROR]: Paper version: git-MultiPaper-"3454702" (MC: 1.18.1)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Server thread dump (Look for plugins here before reporting to Paper!):
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Server thread
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 27 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.FileDispatcherImpl.write0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:62)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.tryWrite(NioSocketImpl.java:398)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.implWrite(NioSocketImpl.java:413)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.write(NioSocketImpl.java:440)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl$2.write(NioSocketImpl.java:826)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.net.Socket$SocketOutputStream.write(Socket.java:1035)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.io.OutputStream.write(OutputStream.java:127)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperConnection.send(MultiPaperConnection.java:79)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.DataOutputSender.send(DataOutputSender.java:27)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperConnection.writeChunk(MultiPaperConnection.java:663)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperIO.scheduleSave(MultiPaperIO.java:34)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		com.destroystokyo.paper.io.chunk.ChunkSaveTask.complete(ChunkSaveTask.java:102)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		com.destroystokyo.paper.io.chunk.ChunkSaveTask.run(ChunkSaveTask.java:63)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		com.destroystokyo.paper.io.chunk.ChunkTaskManager.pollNextChunkTask(ChunkTaskManager.java:231)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.level.ServerChunkCache$MainThreadExecutor.pollTask(ServerChunkCache.java:1321)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.level.ServerChunkCache.pollTask(ServerChunkCache.java:838)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1409)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1392)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:133)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1370)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1276)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:323)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer$$Lambda$3962/0x0000000801453978.run(Unknown Source)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Entire Thread Dump:
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Reference Handler
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 2 | Suspended: false | Native: false | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.Reference.processPendingReferences(Reference.java:253)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.Reference$ReferenceHandler.run(Reference.java:215)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Finalizer
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 3 | Suspended: false | Native: false | State: WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Object.wait(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:172)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Signal Dispatcher
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 4 | Suspended: false | Native: false | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Notification Thread
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 10 | Suspended: false | Native: false | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Common-Cleaner
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 11 | Suspended: false | Native: false | State: TIMED_WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Object.wait(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:162)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: DestroyJavaVM
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 13 | Suspended: false | Native: false | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Log4j2-AsyncAppenderEventDispatcher-1-Async
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 19 | Suspended: false | Native: false | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Thread is waiting on monitor(s):
[22:32:47] [Paper Watchdog Thread/ERROR]: 		Locked on:org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.writeToDestination(RollingRandomAccessFileManager.java:147)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		Locked on:org.apache.logging.log4j.core.appender.OutputStreamManager.flushBuffer(OutputStreamManager.java:283)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		Locked on:org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.flush(RollingRandomAccessFileManager.java:173)
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.writeToDestination(RollingRandomAccessFileManager.java:147)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.OutputStreamManager.flushBuffer(OutputStreamManager.java:283)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.flush(RollingRandomAccessFileManager.java:173)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.directEncodeEvent(AbstractOutputStreamAppender.java:199)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.tryAppend(AbstractOutputStreamAppender.java:190)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.append(AbstractOutputStreamAppender.java:181)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.RollingRandomAccessFileAppender.append(RollingRandomAccessFileAppender.java:243)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:161)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:134)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:125)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:89)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.rewrite.RewriteAppender.append(RewriteAppender.java:84)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:161)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:134)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:125)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:89)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.rewrite.RewriteAppender.append(RewriteAppender.java:84)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:161)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:134)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:125)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:89)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.dispatch(AsyncAppenderEventDispatcher.java:119)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.dispatchAll(AsyncAppenderEventDispatcher.java:81)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.run(AsyncAppenderEventDispatcher.java:63)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: MultiPaperConnection Thread
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 21 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Thread is waiting on monitor(s):
[22:32:47] [Paper Watchdog Thread/ERROR]: 		Locked on:java.base@17.0.2/java.io.BufferedInputStream.read(BufferedInputStream.java:263)
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.SocketDispatcher.read0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:47)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.tryRead(NioSocketImpl.java:261)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:312)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:350)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:803)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.net.Socket$SocketInputStream.read(Socket.java:966)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.io.BufferedInputStream.fill(BufferedInputStream.java:244)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.io.BufferedInputStream.read(BufferedInputStream.java:263)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.io.DataInputStream.readInt(DataInputStream.java:393)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperConnection.run(MultiPaperConnection.java:141)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: MultiPaperFileSyncer
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 23 | Suspended: false | Native: false | State: WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.fs.AbstractWatchService.take(AbstractWatchService.java:118)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.methvin.watcher.DirectoryWatcher.watch(DirectoryWatcher.java:231)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperFileSyncer.run(MultiPaperFileSyncer.java:79)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: FileSystemWatchService
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 24 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.fs.LinuxWatchService.poll(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.fs.LinuxWatchService$Poller.run(LinuxWatchService.java:314)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Timer hack thread
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 25 | Suspended: false | Native: false | State: TIMED_WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.sleep(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.Util$6.run(Util.java:664)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Worker-Main-1
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 26 | Suspended: false | Native: false | State: WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Server thread
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 27 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Thread is waiting on monitor(s):
[22:32:47] [Paper Watchdog Thread/ERROR]: 		Locked on:puregero.multipaper.MultiPaperConnection.send(MultiPaperConnection.java:79)
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.FileDispatcherImpl.write0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:62)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.tryWrite(NioSocketImpl.java:398)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.implWrite(NioSocketImpl.java:413)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl.write(NioSocketImpl.java:440)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/sun.nio.ch.NioSocketImpl$2.write(NioSocketImpl.java:826)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.net.Socket$SocketOutputStream.write(Socket.java:1035)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.io.OutputStream.write(OutputStream.java:127)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperConnection.send(MultiPaperConnection.java:79)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.DataOutputSender.send(DataOutputSender.java:27)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperConnection.writeChunk(MultiPaperConnection.java:663)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		puregero.multipaper.MultiPaperIO.scheduleSave(MultiPaperIO.java:34)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		com.destroystokyo.paper.io.chunk.ChunkSaveTask.complete(ChunkSaveTask.java:102)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		com.destroystokyo.paper.io.chunk.ChunkSaveTask.run(ChunkSaveTask.java:63)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		com.destroystokyo.paper.io.chunk.ChunkTaskManager.pollNextChunkTask(ChunkTaskManager.java:231)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.level.ServerChunkCache$MainThreadExecutor.pollTask(ServerChunkCache.java:1321)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.level.ServerChunkCache.pollTask(ServerChunkCache.java:838)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1409)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1392)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:133)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1370)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1276)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:323)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer$$Lambda$3962/0x0000000801453978.run(Unknown Source)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Paper Watchdog Thread
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 35 | Suspended: false | Native: false | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.management@17.0.2/sun.management.ThreadImpl.dumpThreads0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.management@17.0.2/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:521)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.management@17.0.2/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:509)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.spigotmc.WatchdogThread.run(WatchdogThread.java:204)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Server console handler
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 30 | Suspended: false | Native: false | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.io.FileInputStream.read0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.io.FileInputStream.read(FileInputStream.java:228)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.terminal.impl.AbstractPty$PtyInputStream.read(AbstractPty.java:73)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.utils.NonBlockingInputStream.read(NonBlockingInputStream.java:62)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.utils.NonBlocking$NonBlockingInputStreamReader.read(NonBlocking.java:168)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.utils.NonBlockingReader.read(NonBlockingReader.java:57)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.keymap.BindingReader.readCharacter(BindingReader.java:160)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.keymap.BindingReader.readBinding(BindingReader.java:110)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.keymap.BindingReader.readBinding(BindingReader.java:61)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.reader.impl.LineReaderImpl.doReadBinding(LineReaderImpl.java:923)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.reader.impl.LineReaderImpl.readBinding(LineReaderImpl.java:956)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:651)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:468)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecrell.terminalconsole.SimpleTerminalConsole.readCommands(SimpleTerminalConsole.java:158)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecrell.terminalconsole.SimpleTerminalConsole.start(SimpleTerminalConsole.java:141)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.dedicated.DedicatedServer$1.run(DedicatedServer.java:112)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: pool-4-thread-1
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 36 | Suspended: false | Native: false | State: TIMED_WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Netty Epoll Server IO #0
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 39 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait(Native.java:176)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:281)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:351)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Netty Epoll Server IO #1
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 40 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait(Native.java:176)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:281)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:351)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Netty Epoll MultiPaper IO #0
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 41 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait(Native.java:192)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait(Native.java:185)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.epollWaitNoTimerChange(EpollEventLoop.java:290)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:347)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Java2D Disposer
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 42 | Suspended: false | Native: false | State: WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Object.wait(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.desktop@17.0.2/sun.java2d.Disposer.run(Disposer.java:145)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Craft Async Scheduler Management Thread
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 44 | Suspended: false | Native: false | State: WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Netty Epoll Server IO #2
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 45 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait(Native.java:176)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:281)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:351)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: CompletableFutureDelayScheduler
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 51 | Suspended: false | Native: false | State: WAITING
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:47] [Paper Watchdog Thread/ERROR]: Current Thread: Netty Epoll Server IO #3
[22:32:47] [Paper Watchdog Thread/ERROR]: 	PID: 55 | Suspended: false | Native: true | State: RUNNABLE
[22:32:47] [Paper Watchdog Thread/ERROR]: 	Stack:
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait0(Native Method)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.Native.epollWait(Native.java:176)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:281)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:351)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[22:32:47] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[22:32:47] [Paper Watchdog Thread/ERROR]: ------------------------------
[22:32:48] [Paper Watchdog Thread/INFO]: Stopping main thread (Ignore any thread death message you see! - DO NOT REPORT THREAD DEATH TO PAPER)

**Comment by PureGero** [comment]
Duplicate of #103
